### PR TITLE
Public page

### PR DESCRIPTION
**Pull request by ati7697**

### CODE REVIEW

File: resources/views/slideTest.blade.php [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                            current.getBoundingClientRect().width / 2;
+
+                        current.scroll({
+                            left: nextScrollPosition,
+                            behavior: 'smooth',
+                        });
+                    }
+                };
+            }
+        </script>
```
[the remark this answers is not in the document]
Comment (vrodriguero) [reply resources/views/slideTest.blade.php:66]: see https://laravel.com/docs/9.x/blade#the-once-directive

File: routes/web.php [hunk 2]
```diff
@@ -28,12 +33,4 @@
     return Inertia::render('Dashboard');
 })->middleware(['auth', 'verified'])->name('dashboard');
 
-Route::prefix('/courses')->name('courses')->group(function () {
-    Route::middleware('auth')->prefix('/{course}/lessons')->name('.lessons')->group(function () {
-        Route::get('{lesson}', \App\Http\Controllers\Lessons\ShowLessonController::class)->name('.show');
-        Route::post('{lesson}/save', \App\Http\Controllers\Lessons\SaveLessonController::class)->name('.save');
-        Route::get('{lesson}/load', \App\Http\Controllers\Lessons\LoadLessonController::class)->name('.load');
-    });
-});
```
Comment (vrodriguero) [inline routes/web.php:36]: Is there a reason why this was deleted?

**Review by vrodriguero — CHANGES_REQUESTED** [review]
Hey Ati, 
I left a couple of changes requests :) 

Can you also check if we are using all the images?
I feel like some assets are not in use, so they can be deleted. Also, when you open a PR it makes it a bit easier for me if you have a description of what was done in the PR comment.

Even something simple as: 
Summary of changes: 
- implemented home page 

Also if the changes are UI changes it helps to have some screenshots to explain what it is the change and what it looks like.

A thing that I often do is, open the table for files changed in a new window and go to each change I made to ensure I have described them in the PR description. This helps with ensuring that I am not committing changes that are not intentional. 

Let me know if you have any questions. I am happy to go through the PR process with you if you are unsure of what I meant above hehe

**Comment by ati7697** [comment]
> Hey Ati, I left a couple of changes requests :)
> 
> Can you also check if we are using all the images? I feel like some assets are not in use, so they can be deleted. Also, when you open a PR it makes it a bit easier for me if you have a description of what was done in the PR comment.
> 
> Even something simple as: Summary of changes:
> 
> * implemented home page
> 
> Also if the changes are UI changes it helps to have some screenshots to explain what it is the change and what it looks like.
> 
> A thing that I often do is, open the table for files changed in a new window and go to each change I made to ensure I have described them in the PR description. This helps with ensuring that I am not committing changes that are not intentional.
> 
> Let me know if you have any questions. I am happy to go through the PR process with you if you are unsure of what I meant above hehe

**Comment by ati7697** [comment]
you mean before and after changes I take screenshots? and where should I upload it?

File: package-lock.json [hunk 3]
```diff
@@ -3068,4 +3068,4 @@
             "dev": true
         }
     }
-}
+}
```
[the remark this answers is not in the document]
Comment (ati7697) [reply package-lock.json:3071]: okay

File: resources/views/home.blade.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
+@extends('layouts.app')
+<script>
+    function changeBackgroundImage(bgImage) {
+        document.getElementById("bgDiv").style.backgroundImage = "url(" + bgImage + ")";
+    }
+</script>
```
[the remark this answers is not in the document]
Comment (ati7697) [reply resources/views/home.blade.php:6]: done

File: resources/views/home.blade.php [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+@section('content')
+    <div class="flex flex-col navFont w-screen">
+        {{--  gallery --}}
+        @include('homeLayouts.gallery')
```
[the remark this answers is not in the document]
Comment (ati7697) [reply resources/views/home.blade.php:10]: done

File: resources/views/homeLayouts/course.blade.php [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+{{--                <div class="flex md:text-[14px] sm:ml-3 sm:text-[12px]">--}}
+{{--                    <spann class="m-1">--}}
+{{--                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="md:w-6 md:h-6 sm:w-5 sm:h-5">--}}
+{{--                            <path stroke-linecap="round" stroke-linejoin="round" d="M4.26 10.147a60.436 60.436 0 00-.491 6.347A48.627 48.627 0 0112 20.904a48.627 48.627 0 018.232-4.41 60.46 60.46 0 00-.491-6.347m-15.482 0a50.57 50.57 0 00-2.658-.813A59.905 59.905 0 0112 3.493a59.902 59.902 0 0110.399 5.84c-.896.248-1.783.52-2.658.814m-15.482 0A50.697 50.697 0 0112 13.489a50.702 50.702 0 017.74-3.342M6.75 15a.75.75 0 100-1.5.75.75 0 000 1.5zm0 0v-3.675A55.378 55.378 0 0112 8.443m-7.007 11.55A5.981 5.981 0 006.75 15.75v-1.5"></path>--}}
+{{--                        </svg>--}}
+{{--                    </spann>--}}
+{{--                    <span class="mt-1 ">100</span>--}}
+{{--                    <p class="m-1">Students</p>--}}
+{{--                </div>--}}
+{{--            </div>--}}
```
[the remark this answers is not in the document]
Comment (ati7697) [reply resources/views/homeLayouts/course.blade.php:47]: it is showing how many student are taking the course, its small span under the course picture. I wasn't sure if you want to keep it or not so I comment it to show you later but I forgot :D

File: resources/views/slideTest.blade.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (ati7697) [reply resources/views/slideTest.blade.php:66]: no it was just for testing, I forgot to delete that

File: routes/web.php (hunk 2 above)
Comment (ati7697) [reply routes/web.php:36]: Actually, I don't remember why I deleted it

File: resources/views/layouts/app.blade.php [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    <!-- Fonts -->
+
+
+    <!-- Scripts -->
+    @vite(['resources/sass/app.scss', 'resources/js/app.js'])
+</head>
+<body class="bg-gray-900 w-screen">
+        <header class="">
+            @include('layouts.header')
+        </header>
```
[the remark this answers is not in the document]
Comment (ati7697) [reply resources/views/layouts/app.blade.php:26]: done

File: resources/views/layouts/app.blade.php [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        <div class=" w-screen">
+            @yield('content')
+        </div>
+
+        <footer>
+            @include('layouts.footer')
+        </footer>
```
[the remark this answers is not in the document]
Comment (ati7697) [reply resources/views/layouts/app.blade.php:33]: done

**Pull request closed without merging**